### PR TITLE
Add the DC for LLM monitoring

### PR DESCRIPTION
**Pull request by jinsongo**
The PR is to contribute a new DC for LLM monitoring, that needs to work with `https://github.com/gyliu513/langX101/tree/main/otel/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain` currently. Once the instrumentation code is delivered to a formal repository, I will update the reference information in the README.

### CODE REVIEW

**Review by liurui-1 — COMMENTED** [review]
Hi @jinsongo 
This is a very good contribution! I am sorry for late response because a lot of meetings.
There are 2 recommendations for slight changes:
1. We need to use JDK 11 to build this DC. So please change something in the gradle file and doc file according to the comment.
2. You can see in rdb we have put DC implementations in different package/folders. For example Informix is in impl/informix package/folder. Please use this style to make slight change.

File: ai/build.gradle [hunk 1]
```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group = "com.instana.dc"
+version = "0.0.1"
+
```
[the remark this answers is not in the document]
Comment (jinsongo) [reply ai/build.gradle:8]: Yes, thanks.

File: ai/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+**[Contributing](CONTRIBUTING.md)** |
+**[License](LICENSE)**
+
+---
+ODCAI (OTel based Data Collector for AI) is the tool or template to generate OpenTelemetry metrics for various
+LLM and LLM Applications. All implementation are based on predefined OpenTelemetry Semantic Conventions.
+
+## Requirements
+
+- Java 8+
```
[the remark this answers is not in the document]
Comment (jinsongo) [reply ai/README.md:15]: Yes, thanks.

**Comment by liurui-1** [comment]
> @liurui-1 The LLMDc.java is the specific implementation to receive metrics from instrumentation, it will be extended or more XXXDc.java will be added later. However, the LLMDc.java is a little confusing, not like a specific ai platform, so that could be renamed, but currently it's not very clear about the details for more ai platforms support. I will continue improving.

Thanks @jinsongo ,
Just simply put the implementation code in a sub-directory with any name (for example "openai" or "watsonx")  under "impl". So that other contributors can provide different implementations with your proposed Semantic Convention.

**Review by liurui-1 — APPROVED** [review]

**Pull request merged**